src/subs.f90

```diff
@@ -1083,15 +1217,11 @@ subroutine directory
 #else 
     error stop "Compiler not supported!"
 #endif
-            if(.not. dataExists)then
-                call create_directory("", dataExists, "", .false.)
-                call create_directory("jmean/", jmeanExists, "data/", .false.)
-                call create_directory("deposit/", depositExists, "data/", .false.)
-                call create_directory("detectors/", detectorsExists, "data/", .true.)
-            else
-                call create_directory("jmean/", jmeanExists, "data/", .true.)
-                call create_directory("deposit/", depositExists, "data/", .true.)
-                call create_directory("detectors/", detectorsExists, "data/", .true.)
+            if(.not. dirExists)then
+                mkdirCMD = "mkdir -p "//trim(fileplace)
+                call execute_command_line(mkdirCMD)
+                mkdirCMD = "mkdir -p "//trim(fileplace)//"phasor/"
+                call execute_command_line(mkdirCMD)
```

Comment:
Remove your change and add in my original. think I added this after you started using the code. So add the following line:

```
                call create_directory("phasor/", phasorExists, "data/", .true.)
```

Then add a logical variable call phasorExists